app/about/the-team/page.tsx

```diff
@@ -64,14 +64,30 @@ const TeamMemberInfoCard = (props: TeamMemberInfoCardProps) => {
   )
 }
 
+interface TeamCardHeaderProps {
+  label: string;
+}
+
+const TeamCardHeader = (props: TeamCardHeaderProps) => {
+  if(props.label === 'ArboTracker Collaborating Partners') {
+    return <h2 className="my-4 pb-2 border-b border-background"> {props.label} </h2>;
+  }
+
+  if(props.label === 'SeroTracker Alumni') {
+    return <h3 className="my-4 border-b mt-32 border-background"> {props.label} </h3>;
+  }
+
+  return <h3 className="my-4 border-b border-background"> {props.label} </h3>;
+}
```

Comment:
Yeah I know this is gross and I'm tying the names in Airtable to our frontend. I think this is fine because I don't know if we're going to want to do this kind of thing often enough and keep this change permanently enough for me to do something more fancy and time consuming like putting a new field in Airtable.

This is fine imo, a bit silly but pretty low consequences on a piece of code that might die at any time.

Reply:
Agreed! could we add that in a comment in the code directly so if we ever come back to it we know hwy it does not need to be improved

Reply:
For sure, let me make a comment on my next PR, I've got a couple things I want to hit for Elias in my next one too :)